<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debug</class>
 <widget class="QDialog" name="debug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/SR_icon.png</normaloff>:/SR_icon.png</iconset>
  </property>
  <widget class="QTextEdit" name="debugText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
